docs/ref/request-response.txt

```diff
@@ -998,3 +998,20 @@ Attributes
 .. attribute:: StreamingHttpResponse.streaming
 
     This is always ``True``.
+
+FileResponse objects
+====================
+
+.. versionadded:: 1.8
+
+.. class:: FileResponse
+
+:class:`FileResponse` is a subclass of :class:`StreamingHttpResponse` optimized
+for binary files. It uses ``wsgi.file_wrapper``_ when available.
```

Comment:
Also, this needs to be single backticks around wsgi.file_wrapper for the link to work properly.